simple/stats/db.py

```diff
@@ -487,9 +501,10 @@ def _schema_updates(self) -> None:
     properties_column_exists = rows is not None and len(rows) > 0
     if not properties_column_exists:
       logging.info(
-          "properties column does not exist in the observations table. Altering table to add it."
+          "properties column does not exist in the observations table. Altering table to add all property columns."
```

Comment:
consider spelling out all the columns that are being added to the log statement ("all property columns" is vague)

Reply:
Done.